components/Cta/Cta.styles.ts

```diff
@@ -108,4 +116,4 @@ export const ctaAligns = {
   right: 'su-text-right ml-auto mr-0',
 };
 
-export const ctaGroup = 'list-unstyled gap-x-08em gap-y-1em [&_li]:mb-0 [&_a]:text-09em [&_a]:md:text-20 [&_a]:p-07em [&_a]:md:pt-11 [&_a]:md:pb-12 [&_a]:md:px-30';
+export const ctaGroup = 'list-unstyled gap-x-08em gap-y-1em [&_li]:mb-0 [&_a]:text-09em [&_a]:md:text-20 [&_.cta-button]:p-07em [&_.cta-button]:md:pt-11 [&_.cta-button]:md:pb-12 [&_.cta-button]:md:px-30';
```

Comment:
Only add padding when button links are added to CTA group. We don't want extra padding when they're text links
https://deploy-preview-493--adapt-giving.netlify.app/stories